client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -87,103 +78,55 @@ export class PlanUpgradeUpsell extends PureComponent {
 								) }
 							</b>
 						</p>
 						<p>
 							{ translate(
-								'Why? Based on their research, 50% of the people visiting your site decide to leave or stay within the first three seconds.'
-							) }
-						</p>
-						<p>
-							{ translate( '{{i}}Three seconds!{{/i}}', {
-								components: { i: <i /> },
-							} ) }
-						</p>
-						<p>
-							{ translate(
-								"Wouldn't you like to make a great first impression in those three seconds?"
-							) }
-						</p>
-						<p>{ translate( "Thankfully, there's a way." ) }</p>
-						<p>
-							{ translate(
-								'Great looking sites {{b}}always{{/b}} create great first impressions and leave people wanting to know more about you.',
-								{
-									components: { b: <b /> },
-								}
-							) }
-						</p>
-						<p>
-							{ translate(
-								"That's exactly why we've partnered with some of the world's greatest designers to offer high-end designs that you can use to make your site look incredible."
-							) }
-						</p>
-						<p>
-							{ translate(
-								'These premium themes are beautiful and optimized for mobile and search engines. Most importantly, they are ready to use regardless of your goals.'
-							) }
-						</p>
-						<p>
-							{ translate(
-								"From small businesses to blogs, wedding sites to designer portfolios, you'll find the perfect theme for your needs."
-							) }
-						</p>
-						<p>
-							{ translate(
-								'Typically, this type of high-end WordPress theme {{b}}costs an average of %(premiumThemePriceLow)s, with some going as high as %(premiumThemePriceHigh)s and more{{/b}}.',
-								{
-									args: {
-										premiumThemePriceLow: formatCurrency( premiumThemePriceLow, currencyCode, {
-											precision: 0,
-										} ),
-										premiumThemePriceHigh: formatCurrency( premiumThemePriceHigh, currencyCode, {
-											precision: 0,
-										} ),
-									},
-									components: { b: <b /> },
-								}
-							) }
-						</p>
-						<p>
-							{ translate(
-								'But if you upgrade to a Premium plan with this special offer, you will get our full collection premium themes for just an additional %(discountPrice)s!',
-								{
-									args: {
-										discountPrice: formatCurrency( planDiscountedRawPrice, currencyCode, {
-											stripZeros: true,
-										} ),
-									},
-								}
+								'If you’re familiar with WordPress plugins, you know why that’s exciting. Installing plugins on your site is like downloading an app on your phone. Except, instead of getting a new game or productivity tool, you get new features and functionality for your site.'
 							) }
 						</p>
 						<p>
 							{ translate(
-								"You'll also gain access to some of the most powerful features on WordPress.com:"
+								'There are more than 50,000 WordPress plugins available to help turn your site into any powerful platform you imagine.'
```

Comment:
I'd prefer that we remove the `translate()` calls here. Another way to do it is to keep the translate() calls for now (since it might be too many changges) but ping @Automattic/i18n and request them to exclude this file from the weekly kit to the vendor.

Reply:
Good point! I'll ping them when shipping this to make sure this file is excluded 👍

Reply:
You can just search for it on translate.wordpress.com and mark it as low priority and we won't send it.